include/eve/module/core/regular/impl/simd_cast.hpp

```diff
@@ -34,7 +34,7 @@ simd_cast_(EVE_REQUIRES(cpu_), O const&, T x, as<Target> tgt) noexcept
 
     return res;
   }
-  else if constexpr( sizeof(T) < sizeof(Target) ) { return simd_cast(eve::combine(x, T{}), tgt); }
+  else if constexpr( sizeof(T) < sizeof(Target) ) { return simd_cast(combine(current_api, x, T{}), tgt); }
```

Comment:
why ?

Reply:
Some impls included the `detail/function` header but used the callable version defined in `core/regular` (because the former included the later, which is not the case anymore).
Do you think it would be better to use the callable directly when possible ?